recipes/packer_pipeline_node.rb

```diff
@@ -34,11 +34,41 @@
   group 'alfred'
 end
 
+# put the key associated with the openstack_taster's users on various clusters
+# so that it can access the instances once created
+node.override['osl-jenkins']['secrets_databag'] = 'osl_jenkins'
+node.override['osl-jenkins']['secrets_item'] = 'jenkins1'
+
+openstack_access_keys = credential_secrets['jenkins']['packer_pipeline']
```

Comment:
Also, making the Berksfile point to my local cookbook repos does not work as I don't have copies of munin etc.